docs/faq/loading/synchronize_mysql_into_sr.md

```diff
@@ -0,0 +1,56 @@
+# Synchronize data from MySQL in real time
+
+## What do I do if a Flink job reports an error?
+
+A Flink job reports the error `Could not execute SQL statement. Reason:org.apache.flink.table.api.ValidationException: One or more required options are missing.`
+
+It is possible that the required configuration information is missing in multiple sets of rules, such as `[table-rule.1]` and `[table-rule.2]`, in the SMT configuration file **config_prod.conf**.
```

Comment:
![image](https://user-images.githubusercontent.com/57167462/236394192-95ab7752-deca-4921-85ae-90e03d8b178f.png)
-> A possible reason is that ...?